apps/dot_ext/views/authorization.py

```diff
@@ -7,19 +7,35 @@
 from oauth2_provider.exceptions import OAuthToolkitError
 from oauth2_provider.http import HttpResponseUriRedirect
 from ratelimit.decorators import ratelimit
-from ..forms import AllowForm
+from ..forms import AllowForm, SimpleAllowForm
 from ..models import ExpiresIn
-
+from django.conf import settings
 
 logger = logging.getLogger('hhs_server.%s' % __name__)
 
+__author__ = "Alan Viars"
+
 
 @method_decorator(ratelimit(key='user_or_ip', rate='5/m', method=['GET', 'POST'], block=True), name='dispatch')
 class AuthorizationView(DotAuthorizationView):
     """
     Override the base authorization view from dot to
     use the custom AllowForm.
     """
+    form_class = SimpleAllowForm
+    login_url = getattr(
+        settings, 'AUTHORIZATION_LOGIN_URL', "/mymedicare/login")
+    template_name = getattr(
+        settings, 'AUTHORIZATION_TEMPLATE_NAME', "design_system/authorize.html")
+
+
+@method_decorator(ratelimit(key='user_or_ip', rate='5/m', method=['GET', 'POST'], block=True), name='dispatch')
```

Comment:
Can we use the shared setting here instead of fixing it to 5/m?

Reply:
I agree use the Shared settings for rate (ie. 5/m).
set in base.py and then 
we should define those in an environment settings file.